docs/libcurl/opts/CURLSHOPT_SHARE.3

```diff
@@ -80,7 +81,7 @@ multi handle will share PSL cache by default without using this option.
 All
 .SH EXAMPLE
 .nf
-  CURLSHcode sh
+  CURLSHcode sh;
```

Comment:
This is an unrelated fix which, albeit tiny, should go in separately.

Reply:
Addressed in #8697

Reply:
Ok, thanks!